Maquetado/scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column nowrap;
+  padding: 50px 15px 15px 10px;
+  
+  .text-footer {
+    color: $white-snow;
+    font-family: $font-roboto;
+    font-size: $text-big;
+  }
+  
+  .copy {
+    align-self: flex-end;
+    color: $white-snow;
+    font-size: $text-small;
+    font-weight: normal;
```

Comment:
This is the default value, you can delete it

Reply:
I'm sorry, stupidity sometimes hits me... n_n